src/Domain/CoordinateMaps/TimeDependent/ShapeMapTransitionFunctions/SphereTransition.cpp

```diff
@@ -122,6 +123,7 @@ void SphereTransition::check_magnitudes(const T& mag) const {
           << ". The requested point has magnitude: " << get_element(mag, i));
     }
   }
+#endif
```

Comment:
it's useful to add a closing comment `#endif  // SPECTRE_DEBUG` to these directives for readability. Only change if you get the chance.